packages/common/src/configuration/service-configuration.ts

```diff
@@ -140,7 +140,7 @@ export class ServiceConfiguration {
                 },
                 maxWallClockTimeInHours: {
                     format: 'int',
```

Comment:
why 29?

Reply:
revert to initial implementation since batch api that was returning active/running task issue is fixed